uwsdora.module

```diff
@@ -47,6 +47,18 @@ function uwsdora_ir_citationcmodel_islandora_view_object_alter(AbstractObject &$
     )
   );
   $output['citation.tab']['google_scholar_search']['#markup'] = $link;
+
+  // Adding PDF Download Link
+  $default_viewer = variable_get('islandora_scholar_viewers', array('default' => 'none',));
+  $default_viewer = $default_viewer['default'];
+  if (isset($object['PDF']) && islandora_datastream_access(ISLANDORA_VIEW_OBJECTS, $object['PDF']) && ($default_viewer == 'none') && !isset($output['citation.tab']['pdf_download']['#markup'])) {
```

Comment:
* I don't think wrapping $default_viewer == 'none' in parentheses is necessary.
* For readability sake, it make help to break the if condition into separate lines 
```php
if (isset($object['PDF']) && islandora_datastream_access(ISLANDORA_VIEW_OBJECTS, $object['PDF']) && 
  ($default_viewer == 'none') && 
  !isset($output['citation.tab']['pdf_download']['#markup'])
) {
```

Reply:
Implemented readability feedback